Mailbox.Tests/DataLoaderTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class DataLoaderTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void Load_StreamIsNull_ThrowException()
+        {
+            DataLoader dataLoader = new DataLoader(null);
+        }
+        [TestMethod]
+        public void DataLoader_Save_Success()
+        {
+            List<Mailbox> boxList = new List<Mailbox>();
+            MemoryStream memoryStream = new MemoryStream();
+            DataLoader dataLoader = new DataLoader(memoryStream);
+            boxList.Add(new Mailbox(Sizes.Small, new ValueTuple<int, int>(1, 1), new Person("New", "Person")));
```

Comment:
ValueTuple syntax can be simplified
```suggestion
            boxList.Add(new Mailbox(Sizes.Small, (1, 1), new Person("New", "Person")));
```